qa/suites/upgrade/reef-x/parallel/1-tasks.yaml

```diff
@@ -1,33 +1,33 @@
 tasks:
 - install:
-    branch: pacific
+    branch: reef
```

Comment:
Nit: This commit's title says "bring qa/suites/upgrade/quincy-x".

Reply:
Thanks! Fixed.